src/GraphQL/Resolvers/SourceStreamMethodResolver.cs

```diff
@@ -36,64 +36,113 @@ public SourceStreamMethodResolver(MethodInfo methodInfo, LambdaExpression instan
         /// </summary>
         protected virtual Func<IResolveFieldContext, ValueTask<IObservable<object?>>> BuildSourceStreamResolver(ParameterExpression resolveFieldContextParameter, Expression bodyExpression)
```

Comment:
Looks better. I think after mentioned new PR with observable adapter all if-else blocks here could be flattened.

Reply:
If you mean to remove the 'else' in each of the 'else if' statements, then I would not.  I'm not sure what else you could mean.  It's about as clean as it can get, and it has comments to make it extremely clear which code is matching on what types.  😕 

I could move more code into local functions, but each code block is only about 3-4 lines of code, and doing so might make it even harder to follow.

Let me know if there's something else you suggest I try.

Reply:
I will show later, not in this PR.